api_views/books.py

```diff
@@ -77,4 +84,28 @@ def get_by_title(book_title):
                 }
                 return Response(json.dumps(responseObject), 200, mimetype="application/json")
             else:
-                return Response(error_message_helper("Book not found!"), 404, mimetype="application/json")
\ No newline at end of file
+                return Response(error_message_helper("Book not found!"), 404, mimetype="application/json")
+
+
+def delete_book(book_id):
+    req_data = request.json()
+    try:
+        jsonschema.validate(req_data, delete_book_schema)
+    except:
```

Comment:
Bare except clause is too broad. Specify the expected JsonSchemaValidationError exception and handle specific validation errors